src/Surfnet/StepupRa/RaBundle/Service/RaListingService.php

```diff
@@ -78,4 +79,26 @@ public function searchBy($identityId, $institution)
         $query->setIdentityId($identityId);
         return $this->raListingService->search($query);
     }
+
+    /**
+     * Find all institutions where users of the given institution are RA(A)
+     *
+     * @param string $institution the institution the search should be based on
+     * @return InstitutionListingCollection
+     */
+    public function searchRasInOtherInstitutions($institution)
+    {
+        $query = new RaListingSearchQuery($institution, 1);
```

Comment:
It's better to create a custom query for this and also in MW because it's not the same. This in order to keep the code maintainable and to prevent side effects when changing code when updating something regarding the ra listing logic.

Reply:
This also now only takes page 1 into account.

Reply:
Yes, you are correct.

We can either remove this feature, or create the additional endpoint. As discussed during the stand up yesterday, this feature should be achieved a minimal effort. So removing seems the right choice here.